### ISSUE TITLE
Release BungeeTabListPlus on Modrinth

### ISSUE THREAD

**Issue by Peter-Crawley** [CodeCrafter47/BungeeTabListPlus#707]

**Comment by CodeCrafter47** [comment]
What is Modrinth?

**Comment by Peter-Crawley** [comment]
Kinda like Curseforge, but focused specifically on Minecraft, and generally better in a number of ways: https://modrinth.com/

**Comment by Andre601** [comment]
Yeah. Modrinth originally was a mod sharing platform, but has since expanded to plugins, resource packs, shaders and data packs.

I use it myself and there exist things like a gh action to publish updates.
